bears/natural_language/SpellCheckLintBear.py

```diff
@@ -0,0 +1,25 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='scspell',
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(?P<filename>.*):(?P<line>.\d*):\s*(?P<message>.*)')
+class SpellCheckLintBear:
+    """
+    Lints files to check for incorrect spellings using ``scspell``.
+
+    See <https://pypi.python.org/pypi/scspell> for more information.
+    """
+    LANGUAGES = {"Natural Language"}
+    REQUIREMENTS = {PipRequirement('scspell3k', '2.0')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Spelling'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file):
+        args = ('--report-only', filename,)
```

Comment:
Looks like `scspell` takes in a few useful arguments like dictionary and escaping (https://github.com/myint/scspell).